plugins/node.d/icinga_checks

```diff
@@ -0,0 +1,113 @@
+#!/bin/sh
+
+: << =cut
+
+=head1 NAME
+
+icinga_checks - Plugin to monitor results of icinga monitoring
+
+=head1 CONFIGURATION
+
+No configuration
+
+=head1 AUTHOR
+
+mafri with a lot of help by sumpfralle 
+
+=head1 LICENSE
+
+GPLv3
+
+=head1 MAGIC MARKERS
+
+ #%# family=auto
+ #%# capabilities=autoconf
+
+=cut
+
+ICINGACLI=${ICINGACLI:-$(command -v icingacli)}
+JQ=${JQ:-$(command -v  jq)}
+
+
+if [ "$1" = "autoconf" ] ; then
+    if [ ! -x "$ICINGACLI" ]; then
+        echo "no (could not find 'icingacli')"
+    elif [ ! -x "$JQ" ]; then
+        echo "no (could not find 'jq')"
+    else
+        echo "yes"
+    fi
+    exit
+fi
+
+set -e
+
+if [ "$1" = "config" ]; then
+
+    echo "multigraph icinga_host_checks"
+    echo "graph_title Icinga Host Checks"
+    echo 'graph_args --base 1000'
+    echo 'graph_vlabel Count'
+    echo 'graph_category icinga'
+    echo "up.label Up"
+    echo "down.label Down"
+    echo "unreachable.label Unreachable"
+    echo "pending.label Pending"
+    echo "up.draw AREA"    
+    echo "down.draw STACK"
+    echo "unreachable.draw STACK"
+    echo "pending.draw STACK"
+
+
+    echo "multigraph icinga_service_checks"
+    echo "graph_title Icinga Service Checks"
+    echo 'graph_args --base 1000'
+    echo 'graph_vlabel Count'
+    echo 'graph_category icinga'
+    echo "ok.label Ok"
+    echo "warning.label Warning"
+    echo "critical.label Critical"
+    echo "unknown.label Unknown"
+    echo "pending.label Pending"
+    echo "ok.draw AREA"
+    echo "warning.draw STACK"
+    echo "critical.draw STACK"
+    echo "unknown.draw STACK"
+    echo "pending.draw STACK"
+
+    exit
+fi
+
+if [ ! -x "$ICINGACLI" ]; then
+    echo "could not find 'icingacli'"
```

Comment:
Please output these error messages to stderr (`>&2`). Otherwise you would confuse the output parser.